TheIvano-180294/Task07.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+"""Task07.ipynb
+Automatically generated by Colaboratory.
+Original file is located at
+    https://colab.research.google.com/drive/1tV5j-DRcpPtoJGoMj8v2DSqR_9wyXeiE
+**Task 07: Querying RDF(s)**
+"""
+
+!pip
+install
+rdflib
```

Comment:
these three lines won't even work in a notebook

Reply:
I pressed enter by accident smh. I'll fix it